xml/docu_styleguide.language.xml

```diff
@@ -7,12 +7,21 @@
 <sect1 xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="sec-language">
  <title>Language</title>
 
+ <para>
+  We write our documentation in English, using American English as the default
+  variant. This mostly affects spelling of terms. If spellings differences exist
+  between American and British English, use the American English variant. For
+  terms ending in either -<literal>ise</literal> or -<literal>ize</literal>
+  (like <literal>localise</literal>/<literal>localize</literal>), use the
+  -<literal>ize</literal> variant.
```

Comment:
Minor markup change--I'd suggest using emphasis, not literal here; and I'd include the hyphens in the inline markup:

```suggestion
  verbs ending in either <emphasis>-ise</emphasis> or <emphasis>-ize</emphasis>
  (like <emphasis>localise</emphasis>/<emphasis>localize</emphasis>), use the
  <emphasis>-ize</emphasis> variant.
```

Reply:
Has been fixed with https://github.com/SUSE/doc-styleguide/commit/acda60c1b5138dc9 meanwhile. Closing as resolved.